test/lit_tests/lowering/cpujit.mlir

```diff
@@ -36,6 +36,6 @@ module {
 // CHECK-LABEL: @main
 // CHECK-SAME: (%[[ARG0:.+]]: tensor<64xi64>) -> tensor<64xi64> {
 // CHECK-NEXT:    %[[CALL:.+]] = stablehlo.custom_call @enzymexla_compile_cpu(%arg0) 
-// CHECK-SAME: {api_version = 3 : i32, backend_config = "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00", 
+// CHECK-SAME: {api_version = 4 : i32, backend_config = {attr = "\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00\00"},
```

Comment:
@wsmoses this PR does make this change and I am not sure it affects the kernel stuff in any way

Reply:
no, this actually does need to stay the same (see above)